File: InternetBulletin.Web/Models/PostRatingDto.js
/**
 * PostRatingDtoModel class to represent the post rating dto.
 */
class PostRatingDtoModel {

    /**
     * Gets or sets the post id.
     * @type {string}
     */
    postId = "";

    /**
     * Gets or sets the incremented or decremented value
     * @type {boolean}
     */
    isIncrement = false;

    /**
     * Creates an instance of PostRatingDtoModel
     * @param {string} PostId The post id.
     * @param {boolean} IsIncrement The incremented or decremented value.
     */
    constructor(PostId, IsIncrement) {
        this.postId = PostId;
        this.isIncrement = IsIncrement;
    }

}

export default PostRatingDtoModel;

File: Adapters/IBBS.API.Adapters/Models/Posts/PostRatingDTO.cs
namespace IBBS.API.Adapters.Models.Posts;

/// <summary>
/// The PostDomain Ratings DTO.
/// </summary>
public class PostRatingDTO
{
    /// <summary>
    /// Gets or sets the post id.
    /// </summary>
    /// <value>
    /// The post id.
    /// </value>
    public string PostId { get; set; } = string.Empty;

    /// <summary>
    /// Gets or sets the incremented or decremented value
    /// </summary>
    /// <value>
    /// The incremented or decremented value.
    /// </value>
    public bool IsIncrement { get; set; }
}